docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -118,6 +118,18 @@ Follow the instructions below based on the query type:
 10. Click **Save**. To create a monitor, click [Save and Create Monitor](#creating-slo-monitors).
 
 
+## Creating SLOs from Logs, Metrics, Monitors
+
+You can create SLOs directly from your Sumo Logic log search. This allows you to validate queries, quickly create SLOs, and re-use queries from existing dashboard panels.
+
+1. Enter a new **Log search** query (or use an existing one).<br/><img src={useBaseUrl('img/observability/log-search.png')} alt="log search" />
```

Comment:
Search and query are used interchangeably so it's weird to see them together.

I would suggest a Enter a new Log search or use an existing one.